example/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "greentea-client/test_env.h"
+#include "greentea-client/greentea_metrics.h"
+
+int main()
+{
+    const char key[] = "demo_key";
```

Comment:
`const char* const` would probably be better here.

Reply:
Can do `const char *key`. Is related to coding standard?

Reply:
Personal preference. To me `const char* const` is semantically clearer. Using a pointer does introduce an extra level of indirection however, so the array version is probably fine too. The Mbed coding standard doesn't seem to say anything about this so I'll leave it up to your judgement.

Reply:
I've changed it anyway. Using a pointer for a string literal seems quite common too.